package.json

```diff
@@ -3,7 +3,7 @@
   "private": true,
   "scripts": {
     "start": "yarn build && yarn workspace @spotify-backstage/app start",
-    "build": "lerna run build",
+    "build": "lerna run build --since origin/master",
```

Comment:
@Rugvip The point here is, **origin/master** is accessible from branches when you didn't fork the repo, for example: 

- I did a fork from the **spotify/backstage** repo, so when I run `yarn test/ build` it doesn't work, it will consider the 5 packages to run because there's no **origin/master**, just **master**

- For those make just a clone from **spotify/backstage** repo, `yarn test/ build` will work and ignore when there's no changed packages because using the reference **origin/master**. Because of this the CI now is passing when **test** and **build** commands. The workflow just clones it.

Reply:
Problem with `--since` flag for build is that it won't build when it is supposed to. Running build on a fresh clone won't do anything for example, but what you actually want to do is build all packages.

Reply:
Ok, got it! I'll consider it just for tests for now as you said before.